test/src/test/java/org/corfudb/infrastructure/logreplication/LogEntryWriterTest.java

```diff
@@ -36,10 +38,16 @@ public class LogEntryWriterTest extends AbstractViewTest {
     @Before
     public void setUp() {
         corfuRuntime = getDefaultRuntime();
+        // Initialize TableRegistry and register ProtobufSerializer
+        corfuRuntime.getTableRegistry();
         LogReplicationConfig replicationConfig = Mockito.mock(LogReplicationConfig.class);
         metadataManager = Mockito.mock(LogReplicationMetadataManager.class);
         initMocksForMetadataManager();
-        logEntryWriter = new LogEntryWriter(corfuRuntime, replicationConfig, metadataManager);
+        // Mocking steps for initializing LogEntryWriter.
+        LogReplicationConfigManager mockConfigManager = Mockito.mock(LogReplicationConfigManager.class);
+        when(replicationConfig.getConfigManager()).thenReturn(mockConfigManager);
```

Comment:
consider using Mockito.when()... instead of importing statically, to keep it uniform with other usages in this class